src/components/Card.tsx

```diff
@@ -32,7 +32,7 @@ export interface DispatchProps {
 
 type CardProps = StateProps & DispatchProps;
 
-export const Card = React.memo((props: CardProps) => {
+export const Card = (props: CardProps) => {
     if (props.post.images.length === 0) {
```

Comment:
This `if` is not necessary anymore, both branches render the same thing now.

Reply:
ok